crates/rpc/rpc-types/src/lib.rs

```diff
@@ -10,7 +10,7 @@
 //! Provides all relevant types for the various RPC endpoints, grouped by namespace.
 
 mod admin;
-mod eth;
+pub mod eth;
```

Comment:
this is redundant because reexported all.
```suggestion
mod eth;
```

Reply:
fixed